### ISSUE TITLE
508 Office Reported Issue with Crisis Line Modal

### ISSUE THREAD

**Issue by caw310** [department-of-veterans-affairs/vets-design-system-documentation#4572]
## Description
The 508 Office performed an unannounced audit of [va.gov](http://va.gov/) for OMB recently. They came back with a finding that is our responsibility to fix:
VCL modal (global)
Within the “Talk to veterans crisis line” module, Using the arrow or tab key all Jaws users hear is “711 link” and they don’t hear the rest of the visible text “For TTY, call 711 then 988”
It looks to me like the current header does not use our [Crisis Line Modal](https://design.va.gov/storybook/?path=/story/components-va-crisis-line-modal--default). Ideally, it would for the modernized experience. But we'll still need to fix this for the un-modernized experience as well.

## Details
[Slack thread:](https://dsva.slack.com/archives/C08MQHWV6VD/p1753707654147399) 

## Tasks
- [ ] Update the component to address this issue that occurs using Jaws
- [ ] Have an accessibility specialist confirm the fix
- [ ] Publish

## Acceptance Criteria
- [ ] The crisis line modal issue is fixed.

**Comment by humancompanion-usds** [comment]
The immediate fix could follow the tasks above and that would be fine. However, before this issue is closed I'd like us to be using our Crisis Line modal component in the modernized experience. Because the diff here between the current header and our web component is where the bug lived. This has long been fixed in both va-crisis-line-modal and va-telephone. If we're not ready to be using web-components in the current header in any way then we can file a separate issue for the backlog to fix this later.
